vuln/core/48.json

```diff
@@ -4,6 +4,10 @@
   ],
   "vulnerable": "^6.0.0 || ^8.0.0 || ^9.0.0",
   "patched": "^6.14.0 || ^8.11.0 || ^9.10.0",
-  "ref": "https://nodejs.org/en/blog/vulnerability/march-2018-security-releases/",
+  "publish_date": "28-03-2018",
+  "ref": [
+    "https://nodejs.org/en/blog/vulnerability/march-2018-security-releases/"
```

Comment:
Can you use a string here (`"- https://nodejs.org/en/blog/vulnerability/march-2018-security-releases/"`), core vuln format has not been updated yet and CI will not pass.

Reply:
@vdeturckheim Should we also start the process of updating the core vulns? 
Having this out of sync is not really good I think.

Reply:
@pxlpnk if we  change the format on master, that would make this PR mergeable :)

Reply:
@vdeturckheim I understand, but we have a process now for this. 
In total this would take 60 days to get aligned. 
I think I will get to that in the next few weekends to have the data in the same structure for the same types of data. 

Or did I misunderstand something?

Reply:
You're totally right here!